pages/common/audacious.md

```diff
@@ -1,12 +1,16 @@
 # audacious
 
-> An open-source audio player.
+> An open-source audio player. Descendant of XMMS.
```

Comment:
I would recommend using an alternative word to `Descendant' which is more tech friendly and easily understood.

Reply:
Well, originally, it was a fork of Beep Media Player (BMP). BMP was a (GTK 2) fork of (the GTK 1) XMMS and was abandoned in 2005. It changed a lot of XMMS code although it does have some code that reflect its indirect base.

IMO "descendant" is fine, but as an alternative, what could be used, "indirectly based on" maybe?

Reply:
"An open-source audio player inspired by XMMS"

Otherwise "indirectly based on" is also fine